gst/nnstreamer/tensor_filter/tensor_filter.c

```diff
@@ -780,12 +781,16 @@ static GstCaps *
 gst_tensor_filter_fixate_caps (GstBaseTransform * trans,
     GstPadDirection direction, GstCaps * caps, GstCaps * othercaps)
 {
+#if defined(DBG_DEFINED)
   GstTensorFilter *self;
   GstTensorFilterPrivate *priv;
+#endif
   GstCaps *result;
 
+#if defined(DBG_DEFINED)
```

Comment:
How about adding extra checks using `priv` like other functions?
For example, you may reuse line 791
```
if (priv->fw == NULL)
   return NULL;
```

Reply:
In all the other functions, `priv` is used in other places as well.
Updated as per your suggestion.